samigo/samigo-app/src/webapp/jsf/author/editPart.jsp

```diff
@@ -30,16 +30,51 @@
 **********************************************************************************/
 --%>
 -->
+<%
+	String thisId = request.getParameter("panel");
+	if (thisId == null) {
+		thisId = "Main"	+ org.sakaiproject.tool.cover.ToolManager.getCurrentPlacement().getId();
+	}
+%>
   <f:view>
     <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
       <head><%= request.getAttribute("html.head") %>
       <title><h:outputText value="#{authorMessages.create_modify_p}" /></title>
       <!-- AUTHORING -->
       <script src="/samigo-app/js/authoring.js"></script>
       <script src="/library/js/spinner.js"></script>
+      <script>includeWebjarLibrary('select2');</script>
       </head>
       <body onload="<%= request.getAttribute("html.body.onload") %>">
+<script>
+    function clearSelection(selectObject)
+    {
+        for (var i=0; i<selectObject.options.length; i++)
+        {
+            selectObject.options[i].selected=false;
+        }
+        changeSelect(selectObject);
```

Comment:
The function is used before it's defined.

Reply:
Ok